### ISSUE TITLE
BUG reaction name not parsed with xsplot and branch data

### ISSUE THREAD

**Issue by drewejohnson** [CORE-GATECH-GROUP/serpent-tools#387]
## Summary of issue
When reading an xsplot file, the parsing of the branching yields. When encountering a reaction with ``bra_f`` in the name, assuming these are branch fractions, the data is assigned to the metadata dictionary
https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/cce863d28e058299ca840e1ca296e78a3af668e6/serpentTools/parsers/xsplot.py#L90-L93
where ``xsname`` may not exist in `metadata`

## Actual outcome including console output and error traceback if applicable
```python
    self.metadata[xsname].setData(chunk)
KeyError: 'i82206_03c_bra' 
```
## Expected outcome
No such error is raised.

## Versions

Please provide the following:
Reported from a user
* Version  from ``serpentTools.__version__`` 0.9.2
* Python version - ``python --version`` 3.7.3

## Additional information
We don't have a `bra_f` section in our [xsplot data file](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/master/serpentTools/data/plut_xs0.m) nor is it included in our unit test
https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/297412c46694cf10eda5914289103557e49a34a5/tests/test_xsplot.py#L68-L78

**Comment by drewejohnson** [comment]
Looks like the branching fractions are only included if an isotope is used that has energy-dependent branching ratios, which makes sense. But also explains how it may not have been included in the test file.

**Comment by drewejohnson** [comment]
Closed with #388, merged into develop with #408
